README.md

```diff
@@ -14,6 +14,7 @@ To contribute:
 ## List of positions
 | Name | Location | Application Period | Notes  |
 |---|---|---|---|
+| [Bosch](https://www.bosch.com.sg/careers/) | Marymount | Unkown | SW/IoT/Systems Engineer |
```

Comment:
The links directs to the generic careers page for Bosch. Do you have links to specific tech internships that Bosch offers?